cmd/sb/template.html

```diff
@@ -1,141 +1,132 @@
+      }
+      .table-responsive {
+        padding: 0;
+      }
+      tbody tr:hover th {
+        background-color: rgb(236,236,236);
+      }
+      .table td, th {
+        text-align: center;
+      }
+      .table thead tr th {
+        border-top-width: 1px;
+        border-top: 0px;
+      }
+    </style>
+  </head>
+  
+  <body>
+    <nav class="navbar navbar-light" style="background-color: #FF9800;">
+      <a class="navbar-brand" href="#" style="color: white">{{.Homework.Name}} Scoreboard</a>
+    </nav>
+    <br>
+    
+    <div style="padding: 0px 20px">
+      <div class="container-flux table-responsive">
+        <table id="thetable" class="table table-sm table-hover" data-fixed-columns=true data-fixed-number=1>
+          <thead>
+            <tr>
+              <th scope="col">User</th>
+              <th scope="col">Rank</th>
+              <th scope="col">Passed</th>
+              <th scope="col">Time</th>
+              <th scope="col">Penalty</th>
+              {{range $name := .Homework.Cases}}
+              <th>{{$name}}</th>
+              {{end}}
+            </tr>
+          </thead>
+          <tbody>
+            {{range $row := .Rows}}
+            <tr>
+              <th>{{$row.Submission.User}}</th>
+              <td class="center">{{$row.Rank}}</td>
+              <td class="center">{{$row.NumPassed}}</td>
+              <td>
+                {{$row.TotalTime | printf "%.2f"}}
+              </td>
+              {{if gt $row.PenaltyTime 0.0}}<td class="penalty">{{$row.PenaltyTime | printf "%.0f"}}</td>{{else}}<td></td>{{end}}
+              {{range $cell := $row.Cells}}
+              <td class="{{$cell.Class}}" title="{{$cell.Title}}">{{$cell.Value}}</td>
+              {{end}}
+            </tr>
+            {{end}}
+          </tbody>
+        </table>
+      </div>
+    </div>
+    
+    
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
+    
+    <script>
+      var table = document.getElementById("thetable");
+      
+      for (var i = 5; i < table.rows[0].cells.length; i++) {
+        var minimum = 65536;
+        for (var j = 1; j < table.rows.length; j++) {
+          if (!table.rows[j].cells[i].classList.contains("failed") && !table.rows[j].cells[i].classList.contains("empty")) {
+            var tmp = parseFloat(table.rows[j].cells[i].textContent);
```

Comment:
The number in `textContent` here is rounded, and the result may / may not be what you want.

Reply:
I think I'll keep it this way for now. Two decimal places are enough for color display.
And this way users won't see the different colors with the same digits.

Reply:
My concern was about the best time which is visible. 
Anyway, you can change that later.